src/frontend/Sider/Oppgavebenk/Oppgavetabell.tsx

```diff
@@ -28,24 +36,52 @@ export const utledetFolkeregisterIdent = (oppgave: Oppgave) =>
     oppgave.identer?.filter((i) => i.gruppe === IdentGruppe.FOLKEREGISTERIDENT)[0].ident ||
     'Ukjent ident';
 
-const Oppgavetabell: React.FC<Props> = ({ oppgaver }) => {
-    const { sortertListe, settSortering, sortState } = useSorteringState<Oppgave>(oppgaver, {
-        orderBy: 'opprettetTidspunkt',
-        direction: 'ascending',
-    });
+const utledKeyForOrderBy = (oppgaveRequest: OppgaveRequest): keyof Oppgave => {
+    const orderBy = Object.entries(tabellHeaders).find(
+        ([, { orderBy }]) => orderBy && orderBy === oppgaveRequest.orderBy
+    )?.[0] as keyof Oppgave;
+    return orderBy ?? 'opprettetTidspunkt';
+};
```

Comment:
Skjønner ikke helt denne. Hvorfor kan man ikke bare returnere `oppgaveRequest.orderBy || 'opprettetTidspunkt'`?

Reply:
Har gjenbrukt orderBy-navn fra oppgave-api'et. Her er det `OPPRETTET_TIDSPUNKT` som brukes mot api for sortering.
Aksel-tabellen har filtrering på feltnavn på objektet, dvs `opprettetTidspunkt`.

`utledKeyForOrderBy` finner hvilket felt det skal sorteres ut fra sortBy-feltet i OppgaveRequest
Har forenklet det litt med https://github.com/navikt/tilleggsstonader-sak-frontend/pull/340/commits/987ab334dbb4e4fc251bed8950c6d4f9525d58f4